Mage/src/main/java/mage/abilities/effects/common/CreateDelayedTriggeredAbilityEffect.java

```diff
@@ -96,4 +96,9 @@ public CreateDelayedTriggeredAbilityEffect withCopyToPointer(boolean copyToPoint
         this.copyToPointer = copyToPointer;
         return this;
     }
+
+    @Override
+    public CreateDelayedTriggeredAbilityEffect withTargetDescription(String target) {
+        throw new UnsupportedOperationException("Cannot overwrite target descriptions for CreateDelayedTriggeredAbilityEffect (Adjust the inner effects' pointers instead)");
```

Comment:
Why there are so many unsupported places with `withTargetDescription`? It looks like a standard logic and will work with any effects -- xmage create default target pointer for any effect.

<img width="1031" height="123" alt="shot_250719_083234" src="https://github.com/user-attachments/assets/6fbb527f-65ab-46a4-9854-6adf3e6f51be" />

Reply:
I felt like it'd often be wrong to be setting the same description for all inner effects, but I suppose it'll be correct sometimes so I should support it.